torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3136,6 +3136,37 @@ def sample_inputs_linalg_cond(op_info, device, dtype, requires_grad=False, **kwa
     for shape in shapes:
         yield SampleInput(make_arg(shape))
 
+def sample_inputs_linalg_vander(op_info, device, dtype, requires_grad=False, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+
+    # autograd is not supported for inputs with zero number of elements
```

Comment:
Those inputs can be added to the samples on when `requires_grad=False`

Reply:
Actually that's a stale comment. I added an input with shape `()`.